cylc/flow/data_store_mgr.py

```diff
@@ -188,6 +192,30 @@ def task_mean_elapsed_time(tdef):
     return tdef.rtconfig.get('execution time limit', None)
 
 
+def runtime_from_config(rtconfig):
+    """Populate runtime object from config."""
+    try:
+        platform = rtconfig['platform']['name']
+    except (KeyError, TypeError):
+        platform = rtconfig['platform']
+    directives = rtconfig['directives']
+    environment = rtconfig['environment']
+    return PbRuntime(
+        platform=platform,
+        script=rtconfig['script'],
+        init_script=rtconfig['init-script'],
+        env_script=rtconfig['env-script'],
+        err_script=rtconfig['err-script'],
+        exit_script=rtconfig['exit-script'],
+        pre_script=rtconfig['pre-script'],
+        post_script=rtconfig['post-script'],
+        work_sub_dir=rtconfig['work sub-directory'],
+        execution_time_limit=rtconfig['execution time limit'],
+        directives=json.dumps(directives),
+        environment=json.dumps(environment),
+    )
```

Comment:
Isn't `outputs` missing here?

Reply:
Outputs are at the taskProxy level, but could be added here.. They are less static though.